### DISCUSSION TITLE
Supabase Status Page - Subscribing?

### DISCUSSION THREAD

**Discussion by skoshy** [supabase/supabase#366] [Questions] [answered]
Hi! I'd like to somehow subscribe to the [Supabase status page](https://status.supabase.io/) so I can set up a Slack integration to message me if Supabase is having any issues ASAP. Do incidents get cross-posted immediately to a Twitter account? Or is there any other mechanism I can use to subscribe to incidents in a timely way?

**Comment by kiwicopple** [comment, marked as answer]
Hey @skoshy, yes we plan to add subscriptions - we'll roll it out when we add pricing (~Jan). 

This will be a paid feature because Status Page can cost a bit of money (we pay as more people subscribe).

**Reply by skoshy** [reply]
Gotcha! How about automatic posting incidents to a Twitter status account like SupabaseStatus, is that a possibility as well?

**Reply by kiwicopple** [reply]
I'll look into this one @skoshy  - we're trying to automate our support ops as much as possible so we can stay lean. I'll see if there is a good solution available using twitter 👍
